### ISSUE TITLE
Default text for sharing option

### ISSUE THREAD

**Issue by bepolymathe** [jangernert/FeedReader#529]
Hi,

Thank for this great application. Just a suggestion.
It would be great to be able to change the default text that appear in the box of sharing option.

**Comment by jangernert** [comment]
How would you expect that feature to work? Just save and restore the text from last time? I don't think I would like to have a huge section in the settings for this.

**Comment by bil-elmoussaoui** [comment]
@jangernert In that case saving the text should be per plugin?

**Comment by bepolymathe** [comment]
Yes save and restore would be enough. For example, you change the first time to have a text in your own language and you find the same for others sharing.

**Comment by bil-elmoussaoui** [comment]
@bepolymathe If it's only about getting the string translated. it's already the case. Just head off to Transifex and translate Feedreader to your language. Which will be included in the next stable release.

Unless you want to write a different string. Improvements to the currents string are always welcome of course

**Comment by bepolymathe** [comment]
That's what I did! ;)
